src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] I think it would be better to use class-based instead of functional components for this component because it is according to the project requirements.